packages/@lwc/template-compiler/src/__tests__/fixtures/regression/handler-memoization/expected.js

```diff
@@ -40,6 +42,7 @@ function tmpl($api, $cmp, $slotset, $ctx) {
             {
               on: {
                 click: api_bind(task.delete),
+                touchstart: _m2 || ($ctx._m2 = api_bind($cmp.foo)),
               },
```

Comment:
I changed the fixture to include a global listener:

```
<button onclick={task.delete} ontouchstart={foo}></button>
```

Hopefully this is correct. If not, at least I think it's getting close.